examples/android/app/BUILD.bazel

```diff
@@ -0,0 +1,34 @@
+load("@build_bazel_rules_android//android:rules.bzl", "android_binary")
+
+# An app that consumes android-kt deps
+android_binary(
+    name = "app",
+    custom_package = "examples.android.app",
+    incremental_dexing = 0,
```

Comment:
Enabling the incremental dexing seems to reproduce the bug.

Reply:
Ohhhhh!  Interesting. Good to know.

I also found the error in the rules, but it's good to find a replication.

Reply:
That was perfect, thank you. Definite replication. It fails before the final commit in this review is added.